js/app/controllers/calcontroller.js

```diff
@@ -342,7 +342,21 @@ app.controller('CalController', ['$scope', '$rootScope', '$window', 'CalendarSer
 					} else {
 						element.fullCalendar('option', 'aspectRatio', 1.35);
 					}
-				}
+				},
+			    eventRender: function(event, element) {
```

Comment:
please use tabs instead of 4 spaces. This applies here and in some other places in this commit